core/templates/dev/head/pages/load_constants.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function() {
+  var request = new XMLHttpRequest();
+  request.overrideMimeType('application/json');
+  request.open(
+    'GET', GLOBALS.ASSET_DIR_PREFIX + '/assets/constants.json', false);
+  request.send(null);
+  CONSTANTS = JSON.parse(request.responseText);
```

Comment:
Theoretically, there is no need for `var CONSTANTS`, but I have refactored it a little bit so that the IIFE function returns parsed JSON and that value is then assigned to `var CONSTANTS`